### ISSUE TITLE
Add support for secondary UV map on gltf files.

### ISSUE THREAD

**Issue by Aitolda** [kasenvr/project-athena#884]
This is something built into gltf that doesn't seem to be recognized in the high fidelity engine.  Note the "UVmap nodes" using the secondary uvmap.

[What a gltf with multiple UVs looks like after being imported back into blender.](https://i.gyazo.com/94755a1860e478cafedd0f49c67daa6d.png)

**Comment by JulianGro** [comment]
I am fairly sure that I read somewhere that the engine does not support secondary UV maps on any files, not just gltf.

**Comment by Aitolda** [comment]
> I am fairly sure that I read somewhere that the engine does not support secondary UV maps on any files, not just gltf.

It can confirm that it at least supports secondary UV maps on the normal and occlusion channels (possibly emissive) through fbx,  as I used to use this technique back on blender 2.79, to import into High Fidelity, however I haven't been able to replicate this in 2.8 there may still be a way to do it but I know only know how to in 2.79 at the moment.

**Comment by Aitolda** [comment]
Ok.. I have done some further investigation, and it seems like this is actually working on AO maps within GLTFs, however it does not work currently with Normals or Emissive, as no matter what I try, these will default to the first UV map along with the diffuse.  I have a partial theory though.  AO maps do not get put through the principled shader in blender, they are handled separately by create a special grouped output then gltf just knows is the AO channel.  So.. I think there is no issues with the HF engine itself.  I believe we may need a modified version of the principled shader in blender....

**Comment by SilverfishVR** [comment]
I can confirm that AO does work with a secondary UV map, but maybe not in the most obvious way.
If an object has more than one UV, then UV0 is used for everything but the AO texture (and maybe lightmap, but that is not in GLTF) and AO uses UV1. now you can use the UV Map nodes in blender to specify which texture uses which UV, and that is saved in the GLTF files in the `"texCoord"` property as per the spec, but Vircadia does not use it.
So to use it, you have to make sure that that your AO UV map is the second one in the list in blender.

it is a bit weird but it seems that using `"texCoord"` is not required by the standard, for example tree.js does not, but then again, It don't it supports multiple UV's pr. mesh at all I think.

If secondary UV is actually supported on other maps as indicated it could be implemented for GLTF too by reading the `"texCoord"` property, though I think the recommendation is to not use more than 2, babylon.js actually, fails to render if you do and that is one of the most spec compliant engines out there.

**Comment by Aitolda** [comment]
> Hello! Is this still an issue?

Yes.  However to note, this should not being listed as "Optional" as it's an actual feature of the engine that goes back to the HF does.  It just didn't quite get carried over to GLTF.  In order to have "baked lighting" using AO and Emissive maps (and therefore make performant domains) this is 100% necessary.

[Example](https://gyazo.com/6c03fe26751509843b28a573cdebef80)

In the image here you can see that it is working on the AO (Far left and far right tiled "Hi" shadow), but not on emissive or normals.

[json for testing.](https://aitolda.s3.amazonaws.com/test/UVS_2nd_GLTF/2ndUV_Test.json)
